src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -473,7 +477,29 @@ export class ChatService extends Disposable implements IChatService {
 			if (usedSlashCommand?.command) {
 				this._onDidSubmitSlashCommand.fire({ slashCommand: usedSlashCommand.command, sessionId: model.sessionId });
 			}
-			let rawResponse = await provider.provideReply({ session: model.session!, message: resolvedCommand }, progressCallback, token);
+
+			let rawResponse: IChatResponse | null | undefined;
+
+			if ((typeof resolvedCommand === 'string' && typeof message === 'string' && this.chatSlashCommandService.hasCommand(resolvedCommand))) {
+				// contributed slash commands
+				// TODO: spell this out in the UI
+				const history: IChatMessage[] = [];
+				for (const request of model.getRequests()) {
+					if (typeof request.message !== 'string' || !request.response) {
+						continue;
+					}
+					if (isMarkdownString(request.response.response)) {
```

Comment:
The type of this changed, this check will always fail

Reply:
thanks